pkgdb/src/buildenv/assets/activate

```diff
@@ -93,11 +93,11 @@ if [ $flox_env_found -eq 0 ]; then
   # other things) prepending this environment's bin directory to the PATH.
   if [ -d "$FLOX_ENV/etc/profile.d" ]; then
     declare -a _prof_scripts;
-    _prof_scripts=( $(
-      cd "$FLOX_ENV/etc/profile.d";
+    read -r -d '' -a _prof_scripts < <(
```

Comment:
nonblocking question: what was the shellcheck complaint here?

Reply:
It was related to shellcheck wanting to quote the `"$(...)"` invocation with double quotes, which then caused the results to be interpreted as a string rather than an array. Replacing with `read` is a more explicit way of parsing data into an array ... probably should have gone with that in the first place.  😄